performance_prediction/find_mispredicted_models.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+import numpy as np
+import json
+
+scores = pd.read_csv("absolute_errors_all_all.csv")
+scores.rename(columns={scores.columns[0]: "model name"}, inplace=True)
+scores.set_index("model name", inplace=True)
+
+# For each task separately, find the 3 worst performing models
+worst_per_task = {}
+for task in scores.columns:
+    cutoff = sorted(scores[task].dropna())[-3]
```

Comment:
could the cutoff be passed as an arg here in case we want to change this at some point?